<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMultidimensionalFilterParamsGuiComp</class>
 <widget class="QWidget" name="CMultidimensionalFilterParamsGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="FilterWidthLabel">
     <property name="text">
      <string notr="true">Filter Width</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="FilterWidthSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="FilterHeightLabel">
     <property name="text">
      <string notr="true">Filter Height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="FilterHeightSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="FilterWidthValueLabel">
     <property name="text">
      <string notr="true">1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="FilterHeightValueLabel">
     <property name="text">
      <string notr="true">1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="WidthUnitNameLabel">
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="HeightUnitNameLabel">
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
